Sources/Automerge/ActorId.swift

```diff
@@ -1,15 +1,31 @@
 import AutomergeUniffi
+import Foundation
 
 /// The identifier for collaborators contributing to an Automerge document.
 ///
 /// Each separate instance of an Automerge document should have it's own, unique, `ActorId`.
 /// If you create your own `ActorId`, no concurrent changes should ever be made with the same `ActorId`.
 public struct ActorId: Equatable, Hashable, Sendable {
-    var bytes: [UInt8]
+    var data: Data
+
+    // Creates a new, random actor.
+    public init() {
+        self.init(uuid: UUID())
+    }
+
+    // Creates an actor from a UUID.
+    public init(uuid: UUID) {
+        data = withUnsafeBytes(of: uuid.uuid, { Data($0) })
+    }
+
+    // Creates an actor from data.
```

Comment:
Pick a max size - I'm not stressed about how max, but this _should_ have a limit - and add a check, failing the initializer if it's over a max size. You've mentioned you need 32 bytes for your use case. So as a suggestion, how about 128 bytes as a max?